types/braintree/index.d.ts

```diff
@@ -816,6 +827,55 @@ declare namespace braintree {
         | 'VisaCheckoutCard'
         | 'SamsungPayCard';
 
+    /**
+     * Webhooks
+     */
+
+    export class SampleNotification {
+        bt_signature: string;
+        bt_payload: string;
+    }
+
+    export class WebhookNotification {
+        kind: WebhookNotificationKind;
+        subscription?: Subscription;
+        merchantAccount?: MerchantAccount;
+        transaction?: Transaction;
+        dispute?: Dispute;
```

Comment:
Looks like there are some additional kinds of webhook notifications as well that yield additional attributes (https://developers.braintreepayments.com/reference/general/webhooks/overview#notification-kinds). Think this PR is a great start to implementing webhooks though, so it shouldn't be a blocker. Feel free to submit another PR for those if you're up for it!